src/containers/RecentTrades/RecentTrades.pcss

```diff
@@ -64,6 +64,70 @@
       z-index: 9999;
     }
 
+    &__dropdown {
+      .cr-dropdown {
+        height: 34px;
+
+        .dropdown-toggle,
+        .dropdown-toggle:focus,
+        .dropdown-toggle:active {
+          color: var(--primary-cta-color) !important;
```

Comment:
Avoid to use use ```!important``` (use correct nesting to apply styles). We need to override stiles in future.
All places in this file.